NOTICE.md

```diff
@@ -51,3 +53,15 @@ android:
  * android (Apache license)
  * org.mapsforge, LGPL
  * VTM, LGPL
+
+## Data
+
+|source | license | used as default | included in repo |
+|---------|-----------|---------|------|
+|OpenStreetMap data for the road network | [ODBL](https://www.openstreetmap.org/copyright) | yes | yes
+| GTFS NRW | TODO link | no | yes (TODO - is it? I couldn't find it?)
```

Comment:
Yes, I think @michaz was able to move everything in a sample file and did not need any other GTFS

Reply:
Ok, then I will keep it as yes?

Reply:
What data is this? Is it from "Aachener Verkehrsverbund (AVV)" https://transit.land/feed-registry/operators/o-u-dkb?

Reply:
If yes, then it seems to be open data (CC0). 

http://opendata.avv.de/
http://opendata.avv.de/lizenz_und_readme.txt

Reply:
Yes it is open but we can remove it as just an artificial sample file is used. Sorry my wording wasn't clear enough.

Reply:
No worries, I removed it :)